internal/config/config.go

```diff
@@ -0,0 +1,42 @@
+package config
+
+import (
+	"flag"
+	"os"
+	"strings"
+)
+
+// Config хранит все параметры конфигурации приложения
+type Config struct {
+	ServerAddress string // Адрес запуска HTTP-сервера (флаг -a)
+	BaseURL       string // Базовый адрес для сокращенных URL (флаг -b)
+}
+
+// ParseFlags обрабатывает аргументы командной строки и возвращает Config
+func ParseFlags() Config {
```

Comment:
Здесь можно использовать указатель во избежание копирования структуры.
```suggestion
func ParseFlags()   *Config {
```